Client/TeamTalkClassic/TeamTalkDlg.cpp

```diff
@@ -5717,14 +5717,38 @@ void CTeamTalkDlg::OnServerSaveconfiguration()
 void CTeamTalkDlg::OnUpdateAdvancedStoreformove(CCmdUI *pCmdUI)
 {
     pCmdUI->Enable(m_wndTree.GetSelectedUser()>0);
+    int nUserID = m_wndTree.GetSelectedUser();
+    if(m_moveusers.find(nUserID) != m_moveusers.end()) {
+        pCmdUI->SetCheck(true);
+    } else {
+        pCmdUI->SetCheck(false);
+    }
 }
 
 void CTeamTalkDlg::OnAdvancedStoreformove()
 {
     int nMoveUserID = m_wndTree.GetSelectedUser();
-    if (nMoveUserID)
-        m_moveusers.insert(nMoveUserID);
-
+    if(nMoveUserID) {
+        if(m_moveusers.find(nMoveUserID) != m_moveusers.end()) {
+            m_moveusers.erase(nMoveUserID);
+            if(m_xmlSettings.GetEventTTSEvents() & TTS_USER_JOINED) {

```

Comment:
The move action doesn't really have to do with the TTS event TTS_USER_JOINED. There should probably be a new TTS event for this action.

I saw in another PR you made that you use existing TTS events but there should probably be separate. Alternatively we should have a general TTS event for stating menu action.

Reply:
Should I make a new TTS group called "Menu actions"? Then it can be used for move-action and your other PR with mute on/off.